packs/elastic/pack.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name : elastic
```

Comment:
I'd rather prefer to have a pack per product / tool, since in this case, those products don't have much in common besides the parent organization.

I don't think we could re-use any code and this means we would also need to prefix actions with the service name - e.g. `elasticsearch.indices.optimize`.